jobs/build/release-branch/Jenkinsfile

```diff
@@ -0,0 +1,108 @@
+// https://issues.jenkins-ci.org/browse/JENKINS-33511
+def set_workspace() {
+  if(env.WORKSPACE == null) {
+    env.WORKSPACE = pwd()
+  }
+}
+
+def ssh(cmd) {
+  sh "ssh openshiftdevel -- bash -x <<'EOF'\n${cmd}\nEOF"
```

Comment:
You most likely want to copy the way we do it in the other jobs -- see https://github.com/openshift/aos-cd-jobs/blob/master/sjb/actions/script.py#L22